--- /usr/lib/python2.7/dist-packages/youtube_dl_gui/optionsframe.py	2021-03-04 12:35:07.731635857 +0900
+++ /usr/lib/python2.7/dist-packages/youtube_dl_gui/optionsframe.py	2021-03-04 12:37:43.401630988 +0900
@@ -270,6 +270,7 @@
     LOCALE_NAMES = twodict([
         ('ar_SA', 'Arabic'),
         ('en_US', 'English'),
+        ('ja_JP', 'Japanese'),
         ('ko_KR', 'Korean'),
         ('pt_BR', 'Portuguese'),
         ('ru_RU', 'Russian'),
